src/scipp/plt/model.py

```diff
@@ -0,0 +1,97 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+
+from functools import partial
+import uuid
+
+
+def _make_graphviz_digraph(*args, **kwargs):
+    try:
+        from graphviz import Digraph
+    except ImportError:
+        raise RuntimeError(
+            "Failed to import `graphviz`. "
+            "Use `pip install graphviz` (requires installed `graphviz` executable) or "
+            "`conda install -c conda-forge python-graphviz`.")
+    return Digraph(*args, **kwargs)
+
+
+def _add_graph_edges(dot, node, inventory, hide_views):
+    name = str(node.func)
```

Comment:
There is a small risk that there are multiple nodes wrapping the same function. They would get the same name here. Can you use `node.id` to identify nodes and add the name as a label?

Reply:
Thanks for the suggestion. I managed to clean up the graph quite a bit
![Screenshot at 2022-05-23 10-46-32](https://user-images.githubusercontent.com/39047984/169780822-1acce69d-bbe7-44ef-bdce-699ec6850860.png)
But is does add some not-so-pretty code just for that...